File: nvxapp.client/src/nvx/ClientServer-Service/Infrastructure/MyMokeLongJob/Models/my-moke-long-job-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";



export class MyMokeLongJobModel {
  constructor(
    public jobParameter: string = ""
  ) { }
}

export class MyMokeLongJobInModel {
  
}
export class MyMokeLongJobOutModel extends ModelResult {

  public jobId: string = ""

}

File: nvxapp.server.service/ClientServer-Service/infrastructure/MyMokeLongJob/Models/MyMokeLongJobModel.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;

namespace nvxapp.server.service.ClientServer_Service.infrastructure.MyMokeLongJob.Models
{
    
    public class MyMokeLongJobModel
    {
        public string? JobParameter { get; set; }
    }


    public class MyMokeLongJobInModel
    {
    }

    public class MyMokeLongJobOutModel: ModelResult 
    {
        
        public string? JobId { get; set; }
    }
}